src/AudioParam.js

```diff
@@ -28,13 +29,11 @@ class AudioParam extends DspObject {
       },
       set: function(newVal) {
         this._instrinsicValue = newVal
-        this._toConstant()
-        this._scheduled = []
+        this._automationEventList.add(createSetValueAutomationEvent(newVal, this.context.currentTime))
       }
     })
 
-    this._toConstant()
-
+    this._automationEventList = new AutomationEventList(defaultValue)
```

Comment:
Is that possible to make it private?

Reply:
Do you mean making it really private using the `#` instead of the `_` prefix?